org.eclipse.jdt.core.tests.compiler/src/org/eclipse/jdt/core/tests/compiler/regression/BatchCompilerTest.java

```diff
@@ -4826,15 +4815,15 @@ public void test112() throws Exception {
 			"}",
 		},
 		"\"" + OUTPUT_DIR +  File.separator + "X.java\""
-		+ " -1.5 -source 1.5 -d \"" + OUTPUT_DIR + "\"",
+		+ " -1.8 -source 1.8 -d \"" + OUTPUT_DIR + "\"",
 		"",
 		"",
 		true);
-	String expectedOutput = "// Compiled from X.java (version 1.5 : 49.0, super bit)";
+	String expectedOutput = "// Compiled from X.java (version 1.8 : 52.0, super bit)";
 	checkDisassembledClassFile(OUTPUT_DIR + File.separator + "X.class", "X", expectedOutput);
 }
 //https://bugs.eclipse.org/bugs/show_bug.cgi?id=141830
-//compliance 1.6 source 1.3
+//compliance 1.6 source 1.8
 public void test113() throws Exception {
```

Comment:
Many of these are mislabeled. Also they are the same tests - repeatedly testing the same source/compliance levels ?

Reply:
Correct, there were only two different variations of that test. Removed everything else.